src/main/java/org/apache/commons/codec/binary/Base64.java

```diff
@@ -465,6 +647,11 @@ public static boolean isBase64(final byte octet) {
     /**
      * Tests a given byte array to see if it contains only valid characters within the Base64 alphabet. Currently the
      * method treats whitespace as valid.
+     * <p>
+     * Note: this method threats both characters '+' and '/' and  '-' and '_' as valid base64 characters.
```

Comment:
The phrasing is off here: Using "both" implies 2, but you list 4, each with the word 'and'. What do you really mean?